getting-started.md

```diff
@@ -1,68 +1,92 @@
 # Getting started
 
-## How it works ?
+## How does Chopper work?
 
-Due to Dart limitation on Flutter and Web browser, Chopper does not use reflection but code generation with the help of the [build](https://pub.dev/packages/build) and [source\_gen](https://pub.dev/packages/source_gen) packages from the Dart Team.
+Due to limitations to Dart on Flutter and the Web browser, Chopper doesn't use reflection but code generation with the help of the [build](https://pub.dev/packages/build) and [source\_gen](https://pub.dev/packages/source_gen) packages from the Dart Team.
 
 ## Installation
 
-First you need to add `chopper` and `chopper_generator` to your project dependencies.
+Add the `chopper` and the `chopper_generator` packages to your project dependencies.
 
 ```yaml
 # pubspec.yaml
 
 dependencies:
-  chopper: ^2.0.0
+  chopper: ^3.0.7
 
 dev_dependencies:
-  build_runner: ^1.0.0
-  chopper_generator: ^2.0.0
+  build_runner: ^1.0.0 # Use the latest version available
```

Comment:
It is not safe to use the latest version, I think. :)

Reply:
Yeah, I probably should just remove that comment. :)